src/services/dashboards/all-dashboards/modules/AllDashboardsSelectFilter.vue

```diff
@@ -0,0 +1,155 @@
+                           @select="handleSelectViewers"
+        />
+        <p-divider class="divider"
+                   vertical
+        />
+        <span class="filter-header">Scope</span>
+        <p-select-status v-for="(scope, idx) in scopeFilterList"
+                         :key="`scope-${idx}`"
+                         v-model="proxyScopeStatus.name"
+                         class="select-desktop"
+                         :value="scope.name"
+                         @change="handleSelectScope"
+        >
+            {{ scope.label }}
+        </p-select-status>
+        <p-select-dropdown class="select-tablet"
+                           :selected="proxyScopeStatus.name"
+                           :items="scopeFilterList"
+                           style-type="transparent"
+                           @select="handleSelectScope"
+        />
+    </div>
+</template>
+
+<script lang="ts">
+import type { PropType, SetupContext } from 'vue';
+import {
+    reactive, toRefs,
+} from 'vue';
+
+import { PSelectDropdown, PSelectStatus, PDivider } from '@spaceone/design-system';
+
+import { i18n } from '@/translations';
+
+import { useProxyValue } from '@/common/composables/proxy-state';
+
+import type { DashboardFilterType } from '@/services/dashboards/all-dashboards/type';
+
+export default {
+    name: 'AllDashboardsSelectFilter',
+    components: { PSelectStatus, PSelectDropdown, PDivider },
+    props: {
+        viewerStatus: {
+            type: Object as PropType<DashboardFilterType>,
+            default: undefined,
+        },
+        scopeStatus: {
+            type: Object as PropType<DashboardFilterType>,
+            default: undefined,
+        },
+    },
+    setup(props, { emit }: SetupContext) {
+        const state = reactive({
+            viewerFilterList: [
+                {
+                    label: i18n.t('All'),
+                    name: 'ALL',
+                },
+                {
+                    label: i18n.t('Pubilc'),
+                    name: 'PUBLIC',
+                },
+                {
+                    label: i18n.t('Private'),
+                    name: 'PRIVATE',
+                },
+            ],
+            scopeFilterList: [
+                {
+                    label: i18n.t('All'),
+                    name: 'ALL',
+                },
+                {
+                    label: i18n.t('Entire Workspace'),
+                    name: 'DOMAIN',
+                },
+                {
+                    label: i18n.t('Single Project'),
+                    name: 'PROJECT',
+                },
+            ],
+            proxyViewersStatus: useProxyValue('viewerStatus', props, emit),
+            proxyScopeStatus: useProxyValue('scopeStatus', props, emit),
+        });
+
+        const handleSelectViewers = (filterName: string) => {
+            state.proxyViewersStatus = {
+                ...state
+                    .viewerFilterList.find((d) => d.name === filterName),
```

Comment:
suggestion: what about use the found object? I think it would be more efficient :)
So if user select the same item, `update:viewerStatus` event will not be emitted.

Reply:
Applied now!